client/state/data-layer/wpcom/rewind/status/index.js

```diff
@@ -0,0 +1,32 @@
+/**
+ * External dependencies
+ */
+import { noop } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import {
+	REWIND_STATUS_REQUEST,
+} from 'state/action-types';
+import { dispatchRequest } from 'state/data-layer/wpcom-http/utils';
+import { http } from 'state/data-layer/wpcom-http/actions';
+
+const fetchStatus = ( { dispatch }, action ) => {
+	dispatch( http( {
+		method: 'GET',
+		path: `/activity-log/${ action.siteId }/rewind`,
```

Comment:
note that if this is the path we're hitting our file should be `state/data-layer/wpcom/activity-log/rewind`